sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -191,7 +231,9 @@ Data is broken down according to how you set the `first_symbol_in_high_order_bit
 property. By default, data is broken down in little endian order.
 
 This property is not guaranteed to be respected if the pipe is an inter-kernel
-pipe. The compiler is allowed to optimize the pipe if both sides are visible.
+pipe. 
+
+This property will only have an effect when the `protocol` property is *avalon_streaming*.
 
 |`uses_valid`
```

Comment:
We should make it clear that this property is only applicable if the outside world would be the one driving the signal (i.e. on host/IO-to-kernel pipes).